nifi-nar-bundles/nifi-opentsdb-bundle/.gitignore

```diff
@@ -0,0 +1,22 @@
+# Mac
```

Comment:
I'm not sure we want to track this file, there's a top-level `.gitignore` that should suffice.